packages/instrument-utils/src/guards/isBankInstrument/isBankInstrument.ts

```diff
@@ -1,14 +1,14 @@
-import { BankInstrument, PaymentInstrument } from '@bigcommerce/checkout-sdk';
+import { AchInstrument, BankInstrument, PaymentInstrument } from '@bigcommerce/checkout-sdk';
 
 export function isBankAccountInstrument(
     instrument?: PaymentInstrument,
-): instrument is BankInstrument {
+): instrument is BankInstrument | AchInstrument {
     return instrument?.type === 'bank';
 }
 
 export default function assertIsBankInstrument(
     instrument?: PaymentInstrument,
-): asserts instrument is BankInstrument {
+): asserts instrument is BankInstrument | AchInstrument {
```

Comment:
Can we create a new assertion for `BankInstrument | AchInstrument ` and keep the old `isBankAccountInstrument` as it was?

Reply:
I see, seems like AchInstrument is a type of BankInstrument, is that correct? In that case can we rename this to `assertIsBankTypeInstrument `

Reply:
@animesh1987 Not quite. There is no `iban` field in `AchInstrument`

Reply:
Well in that case they see to be two different things? So probably makes sense to have a separate assert as Peng mentioned above.